appcelerator/titanium_mobile
# TIMOB-5424: Android: Add missing parameters to the ti.start

http://jira.appcelerator.org/browse/TIMOB-5424

Testing steps:
1. Run any app with this change
2. Look in logcat and wait until you see something like "Sending 1 analytics events."
3. Verify with cloud team (I checked with Ronen) that they received the ti.start ping in production with the platform and model fields
